docs/dialect/graphblas_dialect_tutorials/graphblas_lower/graphblas_matrix_reduce_to_scalar.ipynb

```diff
@@ -0,0 +1,341 @@
+{
+ "cells": [
```

Comment:
I feel like this whole notebook could be a removed and a short explanation of why this fused operation exists -- to drop a lot of unused code. Going through an example to show that the function works as expected doesn't add any additional information. Telling them it is equivalent (but more efficient) than `matrix_multiply` followed by `matrix_reduce_to_scalar` is sufficient.

You can also tell them that normally users don't call this directly, but the optimizer will find and convert candidate sequences of graphblas calls automatically.

Reply:
https://github.com/metagraph-dev/mlir-graphblas/pull/85/commits/9799d7fb69b4a15d050b93168a2ac61ba1ea06e3 implements this suggestion. Let me know if this second draft is any better.